config/client-boot.js

```diff
@@ -0,0 +1,27 @@
+import { h, render } from 'preact';
+
+if (process.env.NODE_ENV === 'development') {
+	// enable preact devtools
+	require('preact/debug');
+}
+else if (process.env.ADD_SW && 'serviceWorker' in navigator && location.protocol === 'https:') {
+	// eslint-disable-next-line no-undef
+	navigator.serviceWorker.register(__webpack_public_path__ + 'sw.js');
+}
+
+const interopDefault = m => m && m.default ? m.default : m;
+
+let app = interopDefault(require('app-entry-point'));
```

Comment:
Can this be a normal static import? (Which would make `interopDefault` obsolete as well)

Reply:
No, webpack needs a require() here since it does static analysis during build to find dependencies within `require.ensure()`.